lab/src/Microsoft.ServiceModel.Syndication/src/SyndicationElementExtension.cs

```diff
@@ -175,16 +177,23 @@ public XmlReader GetReader()
                     break;
                 }
                 ++index;
-                reader.Skip();
+
+                await reader.SkipAsync();
             }
+
             return reader;
         }
 
+        public XmlReader GetReader()
+        {
```

Comment:
Do we use this method anywhere? If so, switch the usage to use the async method. This is public so we shouldn't be exposing public sync api's.